packages/wrangler/src/dev/miniflare/index.ts

```diff
@@ -1039,53 +739,28 @@ export function buildMiniflareBindingOptions(
 					)
 				: undefined,
 
-		durableObjects: Object.fromEntries([
-			...internalObjects.map(({ name, class_name: className }) => {
-				return [
-					name,
-					{
-						className,
-						useSQLite: classNameToUseSQLite.get(className),
-						container:
-							config.containerDOClassNames?.size && config.enableContainers
-								? getImageNameFromDOClassName({
-										doClassName: className,
-										containerDOClassNames: config.containerDOClassNames,
-										containerBuildId: config.containerBuildId,
-									})
-								: undefined,
-					},
-				];
-			}),
-			...externalObjects.map(
+		durableObjects: Object.fromEntries(
+			durableObjects.map(
 				({ name, class_name: className, script_name: scriptName }) => {
-					const identifier = getIdentifier(`do_${scriptName}_${className}`);
-					const useSQLite = classNameToUseSQLite.get(className);
-					return config.workerDefinitions === null
-						? [
-								name,
-								{
-									className,
-									scriptName,
-								},
-							]
-						: [
-								name,
-								{
-									className: identifier,
-									scriptName: EXTERNAL_SERVICE_WORKER_NAME,
-									useSQLite,
-									// Matches the unique key Miniflare will generate for this object in
-									// the target session. We need to do this so workerd generates the
-									// same IDs it would if this were part of the same process. workerd
-									// doesn't allow IDs from Durable Objects with different unique keys
-									// to be used with each other.
-									unsafeUniqueKey: `${scriptName}-${className}`,
-								},
-							];
+					return [
+						name,
+						{
+							className,
+							scriptName,
```

Comment:
Before this change, we skipped passing `scriptName` here likely because we consider those to be external DO which would be handled by the dev registry. But that means wrangler dev multi configs mode doesn't support `scriptName`. This fixes that.